plugins/link/SharedMemory.h

```diff
@@ -8,6 +8,8 @@
 
 #include <cstddef>
 
+#include "LinkedMem.h"
```

Comment:
No need to include this here. We can do with a forward declaration within this header.